radicle-httpd/src/api/v1/projects.rs

```diff
@@ -236,10 +253,30 @@ async fn tree_handler(
     let path = path.strip_prefix('/').ok_or(Error::NotFound)?.to_string();
     let storage = &ctx.profile.storage;
     let repo = Repository::open(paths::repository(storage, &project))?;
-    let path = if path.is_empty() { None } else { Some(&path) };
-    let tree = Tree::new(&repo, &sha, path)?;
-    let mut response = json!(tree);
-    response["stats"] = json!(stats(&repo, sha)?);
+    let tree = repo.tree(sha, &path)?;
+
+    let prefix = std::path::Path::new(&path);
+    let entries = tree
+        .entries()
+        .iter()
+        .map(|entry| {
+            json!( {
+                "path": prefix.join(entry.name()),
+                "name": entry.name(),
+                "lastCommit": serde_json::Value::Null,
```

Comment:
This is to match the current unit test. We can replace it with the actual value and update the unit test.

Reply:
Is this comment referring to https://github.com/radicle-dev/heartwood/pull/149/files#diff-85a73839d407aa508b9e78b3756b0e9b6e8d1087a0cafac6a8325a8cf427b65eR266?